tests/cucumber/features/step_definitions/shared_steps.coffee

```diff
@@ -0,0 +1,62 @@
+do ->
+  'use strict'
+
+  _ = require('underscore')
+
+  module.exports = ->
+
+    url = require('url')
+
+    @Given /^I am a new user$/, ->
+      @server.call('reset')
+
+    @When /^I navigate to "([^"]*)"$/, (relativePath, callback) ->
+      @client
+        .url(url.resolve(process.env.ROOT_URL, relativePath))
+        .call(callback)
+
+    registerAccount = (browser, email, callback) ->
+      browser
+        .url(url.resolve(process.env.ROOT_URL, '/'))
+        .click('.sign-in', assert.ifError)
+        .waitForExist('.accounts-modal.modal.in')
+        .click('#at-signUp')
+        .waitForExist('#at-field-password_again')
+        .setValue('#at-field-email', email)
+        .setValue('#at-field-password', 'testuser')
+        .setValue('#at-field-password_again', 'testuser')
+        .submitForm('#at-field-email', assert.ifError)
+        .waitForExist('.sign-out')
+        .call(callback)
+
+    @Given /^I have registered an account$/, (callback) ->
+      registerAccount(@browser, "test@user.com", callback)
```

Comment:
I'm not sure if register account should be a shared step. It might be better to create accounts with fixtures for the tests that aren't specifically testing registration.